packages/opentelemetry-exporter-collector/src/platform/node/CollectorExporterNodeBase.ts

```diff
@@ -33,13 +33,17 @@ export abstract class CollectorExporterNodeBase<
 > {
   DEFAULT_HEADERS: Record<string, string> = {};
   headers: Record<string, string>;
+  keepAlive: boolean = false;
```

Comment:
From my previous comment, i think we should have a `httpAgentOptions` config that just goes straight to the http client if defined.